app/controllers/physical_ticket_controller.rb

```diff
@@ -0,0 +1,10 @@
+class PhysicalTicketController < ApplicationController
+  before_filter :authenticate_user!, except: [:new, :create]
+  load_resource :conference, find_by: :short_title
+  load_and_authorize_resource
+  authorize_resource :conference_registrations, class: Registration
+
+  def new
+  	@unassigned_ticket_purchases = current_user.ticket_purchases.unassigned.by_conference(@conference)
```

Comment:
It was something related to assignment of users to tickets. But I'll remove it now.